test/org/github/jamm/strategies/MemoryMeterStrategyTest.java

```diff
@@ -23,6 +26,17 @@
 @RunWith(Parameterized.class)
 public class MemoryMeterStrategyTest
 {
+    private static final boolean RUNNING_WITH_PRE_JAVA12_JVM = isRunningWithPreJava12Jvm();
```

Comment:
I haven't reviewed these changes as this fails with JDK17 now and I know you were working on it